queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT * FROM animals WHERE EXTRACT(YEAR FROM date_of_birth) BETWEEN 2016 AND 2019;
+SELECT * FROM animals WHERE neutered=true AND escape_attempts<3;
+SELECT date_of_birth FROM animals WHERE name='Agumon' OR name='Pikachu';
+SELECT name,escape_attempts FROM animals WHERE weight_kg>10.5;
+SELECT * FROM animals WHERE neutered=true;
+SELECT * FROM animals WHERE name NOT LIKE '%Gabumon%';
```

Comment:
[OPTIONAL]
- [ ] Nice work writing SQL code for the query **Find all animals not named Gabumon** :clap: However it would be nice to use the **Not Equal to (<>)** instead of **LIKE**.

Reply:
thanks, I will do it

Reply:
@LionRouge1 dONE!